cluster-provision/images/vm-image-builder/customize-image.sh

```diff
@@ -27,6 +32,15 @@ cloud-localds "${CLOUD_INIT_ISO}" "${CLOUD_CONFIG_PATH}"
 echo "Customize image by booting a VM with
  the image and cloud-init disk
  press ctrl+] to exit"
+
+if [[ ${ARCH} = "aarch64" ]]; then
+  readonly DOMAIN_NAME="provision-vm-aarch64"
```

Comment:
Please consider minimizing the arch footprint and change only when needed, 
`DOMAIN_NAME` can stay as is since it just the VM name we spin up and customize and deleted when finished.